src/App.vue

```diff
@@ -1,24 +1,67 @@
 <template>
-  <img
-    alt="Vue logo"
-    src="./assets/logo.png"
-  >
-  <HelloWorld :msg="msg"/>
+  <Suspense>
+    <router-view/>
+  </Suspense>
 </template>
 
 <script setup lang="ts">
-import HelloWorld from './components/HelloWorld.vue';
+import { useRouter } from 'vue-router';
+import { reactive } from 'vue';
 
-const msg = 'Welcome to Your Vue.js + TypeScript App';
+const router = useRouter();
+const state = reactive({ transitionName: 'slide-left' });
```

Comment:
```suggestion
import { reactive } from 'vue';

const data = {
	transitionName: 'slide-left',
};
const state = reactive(data);
```

https://www.notion.so/torderkorea/02-05-Author-Leo-ing-3a408a7676a74460be35f994da657a48?pvs=4#f6485121189f4ae49083c950e6d54e1e

위 링크 참고 하셔요~!!